versions.txt

```diff
@@ -30,7 +30,7 @@ autoinstrumentation-python=0.48b0
 autoinstrumentation-dotnet=1.2.0
 
 # Represents the current release of Go instrumentation.
-autoinstrumentation-go=v0.14.0-alpha
+autoinstrumentation-go=v0.15.0-alpha
```

Comment:
The version of nodejs should be updated as well https://github.com/open-telemetry/opentelemetry-operator/pull/3367